src/main/proto/netflix/titus/titus_job_api.proto

```diff
@@ -176,6 +176,23 @@ message Capacity {
     uint32 desired = 3;
 }
 
+message JobCapacity {
```

Comment:
A comment to help users understand the difference between Capacity and Job Capacity would be useful.